modules/oe_translation_cdt/modules/oe_translation_cdt_mock/src/Plugin/ServiceMock/ServiceMockBase.php

```diff
@@ -103,12 +104,14 @@ protected function needsToken(): bool {
    *   The request parameters in the associative array.
    */
   protected function getRequestParameters(RequestInterface $request): array {
```

Comment:
I've changed it to only look for the parameters inside the endpoint path, while ignoring the base URL.